content/tutorials/60-plugins/60-monitoring/http/index.md

```diff
@@ -72,12 +72,33 @@ This provides us good insight into what the server is actually doing, but doesn'
 
 Usually by using a poll agent like logstash, this takes the json object, transforms it slightly to add some useful meta data and then sends it off to elasticsearch to visualise via Kibana. Keep an eye out for a tutorial  soon!
 
-#### How to install HTTP monitoring?
+Deesptream has two prebuilt services for monitoring.
 
-It's prebuilt into deepstream core, so all you need to do is enable it as follows in your config file:
+### Http monitoring  
+
+Enables an endpoint to get the metrics. After the endpoint is called, metrics are reseted.
+Authentication can be open or setting a `key:value` in the request header.
 
 ```yaml
 monitoring:
   type: http
+  # the endpoint url
+  url: /monitoring,
```

Comment:
I think comas are not actually in the yaml spec, so this might be invalid (I think)

Reply:
yes probably the comma slipped from my json config file